MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java

```diff
@@ -186,13 +184,14 @@ public void onAdsManagerLoaded(AdsManagerLoadedEvent adsManagerLoadedEvent) {
         }
     }
 
+//    if (  )
     // ExoPlayer 2.12+ need this to hook add events
-    public AdErrorEvent.AdErrorListener getAdErrorEventListener() {
+    public AdsImaSDKListener getAdErrorEventListener() {
```

Comment:
this makes sense, I could see this being a problem potentially, I like the change here!

Reply:
However, we might want to consider just making a way to get the AdsImaSDKListener directly, and deprecating these methods so that people should use only that one method. Does that make sense what I'm saying here?

Reply:
It does, this is the actual fix, I will replace these two with single method.

Reply:
Ah ha! that makes sense.

I'd leave these two as is (with the change to the type returned), but add a deprecated annotation to them. In addition, add the new method, and let's change the sample apps to use the new method and I think we're good to go.

Reply:
Ok